README.md

```diff
@@ -1,11 +1,124 @@
-# MSAL Native Auth Sample App Private Preview
+# Sign in users and call a protected web API in iOS (Swift) mobile app by using native authentication
 
-This is a sample application used to demonstrate the CIAM extension of the MSAL framework that is currently in public preview. 
+* [Overview](#overview)
+* [Contents](#contents)
+* [Prerequisites](#prerequisites)
+* [Project setup](#project-setup)
+* [Key concepts](#key-concepts)
+* [Reporting problems](#reporting-problems)
+* [Contributing](#contributing)
 
-## Prerequisite
+## Overview
 
-To install and run the sample application a Mac with Xcode installed is required.
+This sample iOS sample applications demonstrates how to handle sign-up, sign-in, sign out, and reset password scenarios using Microsoft Entra External ID for customers. You can configure the sample to call a protected web API.
```

Comment:
1. Repeated word `sample`.
2. applications is plural

> This sample iOS sample applications

3. sign out and reset password are also nouns, should they be `sign-out`, `reset-password`?

Reply:
I have updated.